account-management-api/src/main/java/uk/gov/di/accountmanagement/lambda/UpdateEmailHandler.java

```diff
@@ -165,9 +165,9 @@ public APIGatewayProxyResponseEvent updateEmailRequestHandler(
 
             auditService.submitAuditEvent(
                     AccountManagementAuditableEvent.UPDATE_EMAIL,
-                    AuditService.UNKNOWN,
+                    PersistentIdHelper.extractPersistentIdFromHeaders(input.getHeaders()),
                     sessionId,
-                    AuditService.UNKNOWN,
+                    internalCommonSubjectIdentifier.getValue(),
```

Comment:
This field also looks wrong to me - we've now got internal subject Id twice. This field is for client id.